frontend/src/components/forms/CreateEmployee.tsx

```diff
@@ -0,0 +1,320 @@
+import React, { useEffect, useState } from "react";
+import {
+  Box,
+  Button,
+  Checkbox,
+  Flex,
+  FormControl,
+  FormErrorMessage,
+  FormLabel,
+  Input,
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalBody,
+  ModalCloseButton,
+  Radio,
+  RadioGroup,
+  Text,
+} from "@chakra-ui/react";
+import { AddIcon } from "@chakra-ui/icons";
+import commonApiClient from "../../APIClients/CommonAPIClient";
+import RoleOptions from "../common/types/Roles";
+import { INVITE_EMPLOYEE_ERROR } from "../../constants/ErrorMessages";
+import CreateToast from "../common/Toasts";
+
+const CreateEmployee = (): React.ReactElement => {
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+  const [isDisabled, setIsDisabled] = useState<boolean>(false);
+  const newToast = CreateToast();
+
+  const [invitedEmail, setInvitedEmail] = useState<string>("");
+  const [invitedFirstName, setInvitedFirstName] = useState<string>("");
+  const [invitedLastName, setInvitedLastName] = useState<string>("");
+  const [invitedAdminStatus, setinvitedAdminStatus] = useState<string>("");
+  const [
+    isTwoFactorAuthenticated,
+    setIsTwoFactorAuthenticated,
+  ] = useState<boolean>(false);
+
+  const [invitedEmailError, setInvitedEmailError] = useState<boolean>(false);
+  const [invitedFirstNameError, setInvitedFirstNameError] = useState<boolean>(
+    false,
+  );
+  const [invitedLastNameError, setInvitedLastNameError] = useState<boolean>(
+    false,
+  );
+  const [
+    invitedAdminStatusError,
+    setInvitedAdminStatusError,
+  ] = useState<boolean>(false);
+
+  useEffect(() => {
+    if (invitedAdminStatus === "1") {
+      setIsDisabled(true);
+    } else {
+      setIsDisabled(false);
+    }
+  }, [invitedAdminStatus]);
+
+  const handleFirstNameChange = (e: { target: { value: unknown } }) => {
+    const inputValue: string = e.target.value as string;
+    setInvitedFirstName(inputValue);
+    if (inputValue) {
+      setInvitedFirstNameError(false);
+    } else {
+      setInvitedFirstNameError(true);
+    }
+  };
+
+  const handleLastNameChange = (e: { target: { value: unknown } }) => {
+    const inputValue: string = e.target.value as string;
+    setInvitedLastName(inputValue);
+    if (inputValue) {
+      setInvitedLastNameError(false);
+    } else {
+      setInvitedLastNameError(true);
+    }
+  };
+
+  const handleEmailChange = (e: { target: { value: unknown } }) => {
```

Comment:
This method is never called, is it disabled for testing?

Reply:
oops I forgot to remove it

Reply:
I fixed the handlers and added them back in as the onchanges

Reply:
wait @Safewaan lowkey we don't need any of the handlers, bc I am alr checking the error in the `handleSubmit` function. and we can just set up the states directly from the input instead of calling the handlers